cmd/kube-proxy/app/options/options.go

```diff
@@ -57,7 +57,6 @@ func NewProxyConfig() *ProxyServerConfig {
 			ResourceContainer:              "/kube-proxy",
 			IPTablesSyncPeriod:             unversioned.Duration{30 * time.Second},
 			UDPIdleTimeout:                 unversioned.Duration{250 * time.Millisecond},
-			Mode:                           componentconfig.ProxyModeUserspace,
```

Comment:
What is wrong in the doc?

On Mon, Feb 1, 2016 at 10:21 PM, Mike Danese <notifications@github.com>
wrote:

> In cmd/kube-proxy/app/options/options.go
> <https://github.com/kubernetes/kubernetes/pull/20464#discussion_r51528627>
> :
>
> > @@ -57,7 +57,6 @@ func NewProxyConfig() *ProxyServerConfig {
> >  			ResourceContainer:              "/kube-proxy",
> >  			IPTablesSyncPeriod:             unversioned.Duration{30 * time.Second},
> >  			UDPIdleTimeout:                 unversioned.Duration{250 * time.Millisecond},
> > -			Mode:                           componentconfig.ProxyModeUserspace,
>
> Ok, it's probably worth adding a BestAvailable ProxyMode here
> <https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/componentconfig/types.go#L65>.
> The doc is already wrong.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20464/files#r51528627>.
>